### PR TITLE
Add `.views` API to handle Crunch views

### PR DESCRIPTION
**Pull request by jjdelc**

### CODE REVIEW

File: scrunch/tests/test_datasets.py [hunk 1]
```diff
@@ -1997,7 +1997,6 @@ def prepare_ds(self):
         dataset_url = 'http://host/api/projects/abc/'
         variables_url = "http://host/api/projects/abc/variables/"
         new_var_url = "http://host/api/projects/abc/variables/123/"
-        hier_url = "http://host/api/projects/abc/variables/hier/"
```
Comment (jjdelc) [inline scrunch/tests/test_datasets.py:2000]: Now accessing this endpoint is lazy, so it is not a request done during initialization. It was here as part of the needed fixtures to init a dataset

File: scrunch/datasets.py [hunk 2]
```diff
@@ -952,11 +953,22 @@ def _reload_variables(self):
...
         self._set_catalog()
         self._vars = self._catalog.index.items()
-        order = self._catalog.hier
+        self._order = None
 
-        # The `order` property, which provides a high-level API for
-        # manipulating the "Hierarchical Order" structure of a Dataset.
-        self.order = DatasetVariablesOrder(self._catalog, order)
+    @property
+    def order(self):
```
Comment (bhagya9631) [inline scrunch/datasets.py:959]: is there a testcase for this?

File: scrunch/views.py [hunk 3]
```diff
@@ -0,0 +1,49 @@
+# coding: utf-8
+
+from scrunch.helpers import shoji_entity_wrapper
+
+
+class DatasetViews:
+    def __init__(self, dataset_resource):
+        self.dataset_resource = dataset_resource
+
+    def create(self, name, columns):
```
Comment (rchacon1) [inline scrunch/views.py:10]: do u plan on adding `rows` at later time?

Comment (rchacon1) [reply scrunch/views.py:10]: and destination `project`?

**Comment by rchacon1** [comment]
@jjdelc can we get the python 2.7 build to pass?

File: scrunch/views.py (hunk 3 above)
Comment (jjdelc) [reply scrunch/views.py:10]: tried to keep it at a minimum this time, just to make sure we had support. I'm sure they'll request it and we can have smaller PRs for those.

**Pull request merged**